scipy/stats/tests/test_hypotests.py

```diff
@@ -348,6 +348,7 @@ def test_continuity(self, kwds, expected, xp):
         xp_assert_close(res.statistic, xp.asarray(expected[0]))
         xp_assert_close(res.pvalue, xp.asarray(expected[1]))
 
+    @pytest.mark.skip_xp_backends('jax.numpy', reason='lazy->no _axis_nan_policy deco')
```

Comment:
Just another one of those failures that needs to be addressed manually until gh-24034 merges.